### PR TITLE
Enable scopes verification for Director

### PR DESCRIPTION
**Pull request by pkosiec**
**Description**

Changes proposed in this pull request:

- Enable scopes verification for Director
- Update Director integration tests for scopes authorization

**Related issue(s)**
#289

### CODE REVIEW

File: tests/end-to-end/run.sh [hunk 1]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# This script is responsible for running Director with PostgreSQL.
```
Comment (aszecowka) [inline tests/end-to-end/run.sh]: is this description correct?

File: tests/end-to-end/director/scopes_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	requiredScopes, err := tc.scopeProvider.GetRequiredScopes("query.applicationsForRuntime")
+	require.NoError(t, err)
+
+	testCases := []struct {
+		Name                 string
+		UseDefaultScopes     bool
+		Scopes               []string
+		ExpectedErrorMessage string
+	}{
+		{Name: "Required Scopes", Scopes: requiredScopes, ExpectedErrorMessage: "runtime does not exist"},
```
Comment (aszecowka) [inline tests/end-to-end/director/scopes_test.go]: this is really strange that in the test for scopes you are expecting error: `runtime does not exist`

File: tests/end-to-end/run.sh (hunk 1 above)
Comment (pkosiec) [reply tests/end-to-end/run.sh]: Nope, fixed

File: tests/end-to-end/README.md [hunk 3]
```diff
@@ -6,5 +6,5 @@ Director folder contains tests of GraphQL API for managing Applications and Runt
 ## Usage
 To run Director tests, execute:
 ```
-env DIRECTOR_GRAPHQL_API={URL} go test -v ./director/...
+./run.sh
```
[the remark this answers is not in the document]
Comment (pkosiec) [reply tests/end-to-end/README.md]: Please run Director first

File: chart/compass/charts/director/values.yaml [hunk 4]
```diff
@@ -6,3 +6,4 @@ deployment:
   securityContext: # Set on container level
     runAsUser: 2000
     allowPrivilegeEscalation: false
+  allowJWTSigningNone: true # If false, tests won't
```
Comment (aszecowka) [inline chart/compass/charts/director/values.yaml]: tests won't what?

**Pull request merged**